### DISCUSSION TITLE
Did muscle-powered vehicles get missed with recent weariness changes?

### DISCUSSION THREAD

**Discussion by natsirt721** [CleverRaven/Cataclysm-DDA#49074] [Q&A] [answered]
Before weariness, fatigue was used to provide penalization when the player undertook strenuous activity. From what I can tell, there has been an effort to, across the codebase, replace the fatigue penalty with weariness. Currently, using a muscle-powered vehicle has a flat activity level of Active, but still causes thirst and fatigue penalties; these penalties are proportional to the load, but retaining them seems to go against the the current trend. Examining the git-blame for the [responsible code](https://github.com/CleverRaven/Cataclysm-DDA/blame/master/src/vehicle.cpp#L4682) indicates that the fatigue penalty has not been changed since weariness was implemented, however [this pull request](https://github.com/CleverRaven/Cataclysm-DDA/pull/47512) removed the fatigue penalty for many other high-activity-level activities.

Questions: 
1. Is it still appropriate for operating a muscle-powered vehicle to directly affect fatigue? 
2. What about thirst?
3. If yes to either of the above, is a discussion about reducing the penalty worthwhile?

Bonus Question: Should the activity level be related to the load? It probably would have little effect currently given the five-minute-chunk assessment of activity level but it looks like there is a push to make that more granular.

**Comment by actual-nh** [comment, marked as answer]
1. Fatigue? No.
2. Probably, at least until a real perspiration system is put in.
3. If it's enough to be noticeable, yes.
Bonus question: It's more granular already, at least in the experimental version.

**Reply by natsirt721** [reply]
Ok, I'm going to try removing the fatigue and thirst penalties and playtest a bit. I would leave thirst in, but it was removed from the other activities so I think consistency beats verisimilitude this time.

**Reply by actual-nh** [reply]
Sounds good; you can mark the above as the answer you've chosen, then?
